### DISCUSSION TITLE
Do you think of the citation as still up-to-date?

### DISCUSSION THREAD

**Discussion by MartinMeilinger** [QUARK-framework/QUARK#59] [General]
In CITATION.cff, there is a list of authors and such.
In that list, Marvin and Jürgen are not listed. Is this (still) intentional?
Also, I'm not sure if the version 1.1.0 still makes sense.

**Comment by Marvmann** [comment]
Hi Martin,

Thanks for the remark. I checked a couple of other open-source libraries and how they cite their work. Apparently, the standard is to cite the authors of the first version and the corresponding publication, if there is any. Since we have published an accompanying paper, I will also add the respective reference.

Besides that, I do not think we should add more authors regularly to avoid confusion about whom to include as contributors.

Let me know what you think. If you agree, I will close the discussion after adding the missing reference to the paper.

**Comment by MartinMeilinger** [comment]
If that's the standard, then it's probably right to do it that way.

**Comment by Marvmann** [comment]
Resolved in #69 and #70
